opensrp-chw-hf/build.gradle

```diff
@@ -178,6 +194,7 @@ dependencies {
     implementation 'com.android.support:multidex:1.0.3'
     //Do not upgrade to 24.jre-1 due to compatibility issues
     implementation 'com.google.guava:guava:20.0'
+    testImplementation project(path: ':opensrp-chw-hf')
```

Comment:
Why is this added? It looks like readding the current code module to be accessible in the tests. Isn't this already accessible

Reply:
Else, this PR looks good

Reply:
previously it was not accessible, ide suggested to add

Reply:
Clean and rebuild your project. I don't see the need to have it here unless the module naming is wrong or the build is corrupted. It was working without this

Reply:
done